src/views/TechnicalAssistance/RequestForm/__snapshots__/Attendees.test.tsx.snap

```diff
@@ -263,7 +263,7 @@ exports[`Trb Request form: Attendees Renders the attendees form 1`] = `
                 value="SYSTEM_MAINTAINER"
               />
               <option
-                label="Contracting Officer's Representative (COR)"
+                label="Contracting Officer’s Representative (COR)"
```

Comment:
What is the purpose of this change? Looks like a quote was replaced with an angled quote?